src/core/qgshtmlutils.h

```diff
@@ -0,0 +1,40 @@
+/***************************************************************************
+                             qgshtmlutils.h
+                             ---------------------------
+    begin                : November 2017
+    copyright            : (C) 2017 by Etienne Trimaille
+    email                : etienne dot trimaille at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSHTMLUTILS_H
+#define QGSHTMLUTILS_H
+
+#include "qgis.h"
+
+/**
+ * \ingroup coregit st
+ * \class QgsHtmlUtils
+ * \brief Class for HTML utilities.
+ * \since QGIS 3.0
```

Comment:
Is this targetting 3.0 or 3.2?

Reply:
It's fixing bug https://issues.qgis.org/issues/17504 so 3.0, Is-it not possible?

BTW, I will fix the `+ * \ingroup coregit st` visible here L24

Reply:
Wasn't sure, if it fixes a bug, 3.0 is the way to go :+1: